cli/installer/README.md

```diff
@@ -34,6 +34,22 @@ Both the PowerShell and Linux/MacOS scripts can be downloaded and executed local
 
 These scripts can be used, for example, to ensure a particular version of azd is installed in a CI/CD environment.
 
+### Download from daily builds 
+
+The `daily` feed is periodically updated with builds from the latest source code in the `main` branch. Use the `version` parameter to download the latest daily release.
```

Comment:
Is this different from running `azd version` after installing from the "daily" feed? 

The "daily" channel is updated every time we merge to `main` with relevant changes today. Relevant changes are based on changed file paths, so updates to, for example, templates wont trigger and release of `azd`. 

We might change things in the future to build periodically instead of on relevant merges to `main`. 

One way we could do this is to have a version table that shows the latest builds from the "daily" feed. We could do this for azd and the vscode extension.

Reply:
Opened an issue to track: https://github.com/Azure/azure-dev/issues/183